Api/Infrastructure/CommandConvention.cs

```diff
@@ -59,10 +59,10 @@ private bool MatchCommandConvention(Type t)
             {
 
             }
-            return (t.Namespace.EndsWith("Messages.Commands")
-                || t.Namespace.EndsWith("MessagingContracts.Commands")
-                || Regex.IsMatch(t.Namespace, @"Messages\..*Commands", RegexOptions.None))
-                && t.Name.EndsWith("Command");
+            return (t.Namespace.EndsWith("Messages.Commands") // Yes
+                || t.Namespace.EndsWith("MessagingContracts.Commands") // No
```

Comment:
Can you explain why this is a `NO`?

Reply:
The conventions don't match perfectly because I didn't clean them up in the example repo. I figured it didn't matter as long as they matched my one command, so I left them mostly as they were in our original code.